infra/repository/github_environment_ci.tf

```diff
@@ -0,0 +1,18 @@
+resource "github_repository_environment" "github_repository_environment_prod_ci" {
+  environment = "prod-ci"
```

Comment:
what is the purpose of this environment?

Reply:
To manage azure login through federated credentials and deployment reviewers